polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -56,6 +53,7 @@ export interface FrameProps {
   onNavigationDismiss?(): void;
   /** A boolean property indicating whether there should be space for a sidebar
    * @default false
+   * @deprecated As of release 12.0
```

Comment:
Not sure we should promise the version this gets removed at. Things are moving quickly and this could get skipped

Reply:
Okay, agreed. Can always come back and add this if we are more confidence in removing it